### PR TITLE
MMDP-506 Manage options in the Staff Strength Dropdown.

### PR DESCRIPTION
**Pull request by Musacoli**
**What does this PR do?**

Implements the ability for an `administrator` to `manage` options in the `Staff Strength` Dropdown.

**Description of Task to be completed?**
- [x] implement `create` Staff Strength endpoint
- [x] implement `update` Staff Strength endpoint
- [x] implement `get` Staff Strength endpoint
- [x] implement `delete` Staff Strength endpoint
- [x] consume `create` Staff Strength endpoint
- [x] consume `update` Staff Strength endpoint
- [x] consume `get` Staff Strength endpoint
- [x] consume `delete` Staff Strength endpoint

**How should this be manually tested?**

- Pull and checkout to`ft-add-dropdown-management-staffstrength-MMDP-506` branch
- run `yarn dev-server` 
- route to `localhost:1234/dropdowns/staff-strength`
- Fill in the form with all required fields and click the save button

**Any background context you want to provide?**

n/a

**What are the relevant Jira issues?**

[MMDP-506](https://viisaus.atlassian.net/browse/MMDP-506)

**Screenshots (if appropriate)**
**With 1 option**
image

**With more added options**
image

**Questions:**
N/A

### CODE REVIEW

File: client/containers/Stakeholders/ManageStaffStrength.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
+import React, { Component } from 'react';
+import { connect } from 'react-redux';
+import PropTypes from 'prop-types';
+import EditDropdownForm from '../../components/Stakeholders/EditDropdownForm';
+import {
+  // createStaffStrengths,
```
Comment (jomasim) [inline client/containers/Stakeholders/ManageStaffStrength.js]: get rid of these commented lines

File: client/containers/Stakeholders/ManageStaffStrength.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+  }
+
+  addInputData = (data) => {
+    const { optionsData } = this.state;
+    this.setState({
+      optionsData: [...optionsData, data],
+    });
+  };
+
+  // loading = () => {
```
Comment (jomasim) [inline client/containers/Stakeholders/ManageStaffStrength.js]: get rid of these commented lines

**Comment by eduhmik** [comment]
> I am running into an error when running the app kindly check and fix it
> Screenshot 2019-03-28 at 12 13 52

Experienced this issue too

**Comment by bozicschucky** [comment]
Hey @Musacoli, the updating doesn't work, it returns that error in the screenshot. 
Try fixing it thanks.
Screenshot 2019-04-01 at 09 59 58

**Comment by bozicschucky** [comment]
When a user post a new drop drown a validation error is shown yet the drop down is created.
Screenshot 2019-04-01 at 09 59 16

**Comment by Musacoli** [comment]
Pull Request was closed by error.

**Pull request merged**